src/transformers/pipelines/image_text_to_text.py

```diff
@@ -57,10 +57,9 @@ def __init__(self, messages: Dict, images: Union[str, List[str], "Image.Image",
         for message in messages:
             if not ("role" in message and "content" in message):
                 raise ValueError("When passing chat dicts as input, each dict must have a 'role' and 'content' key.")
-        images = retrieve_images_in_messages(messages, images)
+        retrieve_images_in_messages(messages, images)
```

Comment:
modifying object inplace is usually harder to debug and can lead to unexpected bugs in the future, I would better avoid it